eks.clusters.cnp.example.org/definition.yaml

```
@@ -0,0 +1,48 @@
+---
+apiVersion: apiextensions.crossplane.io/v1alpha1
+kind: CompositeResourceDefinition
+metadata:
+  name: eks.clusters.cnp.example.org
+spec:
+  defaultCompositionRef:
+    name: eks.clusters.cnp.example.org
+  connectionSecretKeys:
+  - kubeconfig
+  crdSpecTemplate:
+    group: cnp.example.org
+    version: v1alpha1
+    names:
+      kind: EKS
+      listKind: EKSList
+      plural: eks
+      singular: eks
+    validation:
+      openAPIV3Schema:
+        type: object
+        properties:
+          spec:
+            type: object
+            properties:
+              parameters:
+                type: object
+                description: EKS configuration parameters.
+                nodes:
+                  type: object
+                  description: EKS node configuration parameters.
+                  count:
+                    type: int
+                    description: Desired node count, from 1 to 100.
+                  size:
+                    type: string
+                    description: Size of node.
+                    enum:
+                    - small
+                    - medium
+                    - large
+                  required:
+                  - count
+                  - type
```

should this be `size`?
```suggestion
                  - size
```